db/migrate/20231205141210_create_posts.rb

```diff
@@ -0,0 +1,13 @@
+class CreatePosts < ActiveRecord::Migration[7.1]
+  def change
+    create_table :posts do |t|
+      t.references :author, foreign_key: { to_table: :users, name: 'author_id' }
+      t.string :title
+      t.text :text
+      t.integer :commentsCounter, default: 0
+      t.integer :likesCounter, default: 0
```

Comment:
- Hey, the naming convention in ruby uses the snake_case, kindly update the commentCounter, likesCounter to likes_counter and comments_counter.